ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -295,6 +295,10 @@ def pytest_configure(config):
         log.info(
             f"Dump of the consolidated config file is located here: " f"{config_file}"
         )
+        if config.getoption("--reportportal"):
+            set_rp_client_log_level()
+            set_report_portal_config(config)
+
         set_report_portal_config(config)
```

Comment:
Im actually just realizing that we are still calling `set_report_portal_config(config)` here after the `if`. I imagine this is not intentional.

Reply:
Done